smack-core/src/main/java/org/jivesoftware/smack/proxy/ProxyInfo.java

```diff
@@ -24,17 +28,18 @@
  */
 
 public class ProxyInfo {
+
```

Comment:
Please do not include unrelated whitespace changes in a commit. If you really think there should be a newline, then create an extra commit and pull request for this.

Reply:
Ah, my bad. Did not notice the extra line. Reverting that